internal/controller/controllers/agentserviceconfig_controller.go

```diff
@@ -120,8 +123,7 @@ func (r *AgentServiceConfigReconciler) Reconcile(ctx context.Context, req ctrl.R
 	}
 
 	msg := "AgentServiceConfig reconcile completed without error."
-	r.Recorder.Event(instance, "Normal", aiv1beta1.ReasonReconcileSucceeded, msg)
-	conditionsv1.SetStatusConditionNoHeartbeat(&instance.Status.Conditions, conditionsv1.Condition{
+	conditionsv1.SetStatusCondition(&instance.Status.Conditions, conditionsv1.Condition{
```

Comment:
I think this was changed recently to be `NoHeartbeat`, why change it back?

Reply:
Let's just get this in a handle these in a follow-up.

Reply:
Missed it in the rebase. Good eye. I've changed it back.